dsc/docs-conceptual/dsc-2.0/configurations/configDataCredentials.md

```diff
@@ -229,27 +204,21 @@ ModuleVersion = "1.0";
 
 ### Credentials in transit and at rest
 
-- The **PSDscAllowPlainTextPassword** flag allows the compilation of MOF files that contain
-  passwords in clear text. Take precautions when storing MOF files containing clear text passwords.
-- When the MOF file is delivered to a Node in **Push** mode, WinRM encrypts the communication to
-  protect the clear text password unless you override the default with the **AllowUnencrypted**
-  parameter.
-  - Encrypting the MOF with a certificate protects the MOF file at rest before it has been applied
-    to a node.
-- On the Node, MOF files are encrypted at rest.
+The **PSDscAllowPlainTextPassword** flag allows the compilation of MOF files that contain passwords
+in clear text. Take precautions when storing MOF files containing clear text passwords.
```

Comment:
Talk to Michael Greene about this. We may still need to talk about encrypting the MOF. There is history around this point that MG can provide.